WordPress/Classes/ViewRelated/Stats/Extensions/UITableViewCell+Stats.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+extension UITableViewCell {
+
+    func addRows(_ dataRows: [StatsTotalRowData],
+                 toStackView rowsStackView: UIStackView,
+                 limitRowsDisplayed: Bool = true,
+                 rowDelegate: StatsTotalRowDelegate? = nil) {
+
+        let numberOfDataRows = dataRows.count
+
+        if numberOfDataRows == 0 {
```

Comment:
nitpick and my personal style thing, but this could be written as a `guard` to ensure we don't accidentally forget to escape the scope when this gets modified in the future :)

Reply:
Good catch! And done!